elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/validator/DynamicConfigValidator.java

```diff
@@ -296,8 +429,10 @@ private static void printHelp(Options options) {
      * @return Path to model dir
      */
     private String formatClassPath(String filePath) {
-        if (filePath.indexOf(RESOURCES) > -1) {
-            return filePath.substring(filePath.indexOf(RESOURCES) + RESOURCE_LENGTH);
+        if (filePath.indexOf(RESOURCES + File.separator) > -1) {
```

Comment:
Do we have unit tests for this function?

Reply:
Added.